man/file-hierarchy.xml

```diff
@@ -420,6 +420,22 @@
         this directory.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><filename>/sys/fs/cgroup/</filename></term>
+        <listitem><para>A virtual kernel file system exposing process
+        control groups (cgroups). This file system is an API to
+        interface with the kernel and not a place where normal files may
+        be stored. In sandboxed/containerized setups, this directory may
+        either not exist or may include a subset of functionality. On
+        current systems, this directory will refer to the
+        <literal>cgroup2</literal> filesystem, which provides a unified
+        cgroup hierarchy for all resource controllers. On older systems,
+        this directory may refer to the <literal>cgroup</literal> (v1)
+        filesystem, and will contain a subdirectory
```

Comment:
s/will/might/ - systems can be booted in legacy-only mode too

Reply:
Was not aware that was still a supported configuration. :(

What about "should", by way of encouragement without hard requirement?

Reply:
We still document and support all three modes: legacy, hybrid, and unified, see e.g. dicussion of `systemd.legacy_systemd_cgroup_controller` in systemd(1), and https://systemd.io/CGROUP_DELEGATION/#three-different-tree-setups-. And those three modes are confusing for people, so I think we should be careful here do describe the three possibilities correctly.


"will refer to …" → "will contain …", or "… will be mounted on", or maybe "uses …".

Please move the sentence about containers after the description of what happen on "real" systems.

> "On older systems,  this directory may refer to the <literal>cgroup</literal> (v1)"

This is incorrect. It will use tmpfs, with subdirs with cgroup and cgroup2.